RangeRequest.bs

```diff
@@ -10,26 +10,218 @@ Editor: Chris Lilley, W3C, https://svgees.us/, w3cid 1438
+
+<div class=example>
+The use of the <code>incremental</code> keyword in this CSS rule indicates to the browser they should use IFT.
+<pre>
+@font-face {
+    font-family: "MyCoolWebFont";
+    src: url("MyCoolWebFont.otf") format(opentype supports incremental);
+}
+</pre>
+</div>
+
+Note: Each individual <code>@font-face</code> block may or may not opt-in to IFT. This is due to the variety of ways fonts are used on web pages. Authors have control over which fonts they want to use this technology with, and which they do not.
+
+<h2 id="method-selection">
+IFT Method Selection</h2>
+
+When a page has indicated that a particular font should utilize IFT technology, the browser must determine which method to use. Different browsers may support different IFT methods, and different servers may support different IFT methods, so a negotation occurs at such:
+
+1. The browser makes the first request to the server. If the client prefers the <a href="https://w3c.github.io/IFT/Overview.html#patch-incxfer">patch-subset method</a>, it sends the relevant <a href="https://w3c.github.io/IFT/Overview.html#negotiating-transfer-type">query parameter</a>. If the client prefers the range-request method, it does not send the query parameter.
+2. If the server receives the patch-subset query parameter and wishes to honor it, the server may reply with the <a href="https://w3c.github.io/IFT/Overview.html#handling-patch-response">patch-subset magic number</a>. Otherwise, the server may reply with the [[!RFC7233]] <a href="https://httpwg.org/specs/rfc7233.html#header.accept-ranges"><code>Accept-Ranges</code></a> header.
+3. If the client receives the patch-subset magic number, it commences using the patch-subset method. Otherwise, if the client receives the <code>Accept-Ranges: bytes</code> header, it commences using the range-request method. Otherwise, the whole font file is downloaded, and the current non-incremental loading behavior is used.
+
+<h3 id="fallback">
+IFT Method Fallback</h3>
 
 <em>This section is not normative.</em>
 
-Range Request incremental font transfer is an alternate method for incrementally transferring fonts
-that leverages HTTP range requests.
+This summarizes behaviors that result from the above method selection.
+
+<table class="data">
+    <thead>
+        <tr>
+            <th>&nbsp;
+            <th>Client prefers range-request method
+            <th>Client prefers patch-subset method
+    <tbody>
+        <tr><th>Server supports both range-request method and patch-subset method
+            <td>Client makes initial request without query parameter, and possibly with the <code>Range</code> header. Because all patch-subset servers must support the range-request method, the server replies with <code>Accept-Ranges</code> and initial font data. Client/server commence using range-request method.
```

Comment:
Considering the data we collected during PFE evaluation work on how network conditions affect the font transfer efficiency, and assuming that servers supporting both methods would eventually learn how to optimize data transfer based on those conditions - should we allow a server to _suggest_ a recommended approach? E.g. when a client wants to opt in for incremental transfer but the server recommends full font load, or when a client initially opts in for "range-request" but the server recommends "patch-subset" (or vice versa).

Reply:
Interesting. Should probably be broken out into a separate issue

Reply:
See issue #56 for continued discussion.

Reply:
Because we have a dedicated issue for this, I'll resolve this for now, and add an issue in the PR with a link to the issue.